content/20211219-dev-meetups.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Building a software developer community"
+slug: "building_a_software_developer-community"
+cover: "016_dev_meetups.png"
+date: 2021-10-28
```

Comment:
adjust the date here I think?

Reply:
Do we want the original recording date there or the release date?

Reply:
ive been using the record date, so I can see the context of when was td about around that time

usually get the record date from zencastr mp3 creation

Reply:
i think 10-28 sounds right though actually